### DISCUSSION TITLE
When will you  Update the CUDA Version

### DISCUSSION THREAD

**Discussion by CSAnetGmbH** [open-webui/open-webui#4130] [General]
Hello, when will the CUDA Version will Updated to Version 3,10 ? Only the main Version ist updated. So Wathtower doesn´t Update the Cuda Verion, it is still on 3,8

**Comment by CSAnetGmbH** [comment]
i mean the Image wioth the CUDA TAG

**Comment by Meticulous7** [comment]
Would like to know the same. Just noticed how out of date the CUDA image is
